### PR TITLE
Address follow up comments for the multi include guide

### PR DESCRIPTION
**Pull request by yorinasub17**
Follow up to https://github.com/gruntwork-io/gruntwork-io.github.io/pull/665

### CODE REVIEW

**Comment by rhoboat** [comment]
Can you also please fix the link to the parsing order? `https://terragrunt.gruntwork.io/docs/getting-started/configuration/#configuration-parsing-order`

File: _posts/2021-11-23-how-to-dry-your-ref-arch.adoc [hunk 1]
```diff
@@ -584,12 +589,16 @@ dependency "vpc" {
...
-  # This input variable is not defined in the underlying Terraform module. We use _
-  # to decrease the likelihood of accidentally using a defined variable here.
+  # This input variable is for convenience only and is not defined in the underlying Terraform module.
+  # We use _ to mark it as "private" and decrease the likelihood of accidentally using a defined variable here.
   _vpc_id = dependency.vpc.outputs.vpc_id
 }
 ----
 
+You can then access the `_vpc_id` input variable in the child configuration by using an exposed `include` block (under
+the reference, `include.PARENT.inputs._vpc_id`, like so:
```
Comment (rhoboat) [inline _posts/2021-11-23-how-to-dry-your-ref-arch.adoc:599]:
```suggestion
the reference, `include.PARENT.inputs._vpc_id`), like so:
```

File: _posts/2021-11-23-how-to-dry-your-ref-arch.adoc [hunk 2]
```diff
@@ -604,3 +613,10 @@ inputs = {
   }
 }
 ----
+
+Note that since the `dependency` reference is in the `inputs` attribute, you can only reference it in the child
+configuration's `inputs` attribute. You can not configure a `locals` block in the child that accesses the parent
+`inputs` attribute.
```
Comment (rhoboat) [inline _posts/2021-11-23-how-to-dry-your-ref-arch.adoc:619]: ➕ Nice!

File: _posts/2021-11-23-how-to-dry-your-ref-arch.adoc [hunk 3]
```diff
@@ -604,3 +613,10 @@ inputs = {
   }
 }
 ----
+
+Note that since the `dependency` reference is in the `inputs` attribute, you can only reference it in the child
+configuration's `inputs` attribute. You can not configure a `locals` block in the child that accesses the parent
```
Comment (rhoboat) [inline _posts/2021-11-23-how-to-dry-your-ref-arch.adoc:618]: Super NIT

```suggestion
configuration's `inputs` attribute. You cannot configure a `locals` block in the child that accesses the parent
```

**Review by rhoboat — APPROVED** [review]
Thanks for doing this! LGTM!

**Comment by yorinasub17** [comment]
Thanks for rapid feedback! Going to merge this in!

**Pull request merged**